src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -66,6 +66,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
```

Comment:
I'm not sure about the title of this section. I'd see something more specific, such as "Calling the API as a bot" or something like that? To inform that this section doesn't describe bots in general, but tackles a specific scenario.

Also, I think we should link this section [here](https://deploy-preview-1372--livechat-public-docs.netlify.app/messaging/agent-chat-api#methods) 👇
<img width="821" alt="image" src="https://user-images.githubusercontent.com/42515529/236184101-f58c5aaa-745c-42b5-9e36-309e7019f70f.png">

Reply:
Yup, a specific title will be much better now that you have written it.